.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-  "presets": ["es2015", "stage-0", "react"]
+  "presets": ["es2015", "stage-0", "react"],
+  "env": {
+    "development": {
+      "presets": ["react-hmre"]
```

Comment:
not a huge deal, but (and I didn't know about this preset!) this is not maintained any more. Should be ok but is there something else we can swap it out with down the road or at least note that it's something that could be an issue later?